go/ct/driver/probe.go

```diff
@@ -0,0 +1,198 @@
+//
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use
+// of this software will be governed by the GNU Lesser General Public Licence v3
+//
+
+package main
+
+import (
+	"fmt"
+	"runtime"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"pgregory.net/rand"
+
+	"github.com/Fantom-foundation/Tosca/go/ct"
+	"github.com/Fantom-foundation/Tosca/go/ct/common"
+	cliUtils "github.com/Fantom-foundation/Tosca/go/ct/driver/cli"
+	"github.com/Fantom-foundation/Tosca/go/ct/gen"
+	"github.com/Fantom-foundation/Tosca/go/ct/rlz"
+	"github.com/Fantom-foundation/Tosca/go/ct/spc"
+	"github.com/Fantom-foundation/Tosca/go/ct/st"
+	"github.com/dsnet/golib/unitconv"
+	"github.com/urfave/cli/v2"
+	"golang.org/x/exp/maps"
+)
+
+var ProbeCmd = cli.Command{
```

Comment:
Can you wrap the command into the `cliUtils.AddCommonFlags(..)` to add general flag support?